doc/base/classes.xml

```diff
@@ -40647,14 +40659,14 @@
 			<return type="Vector3">
 			</return>
 			<description>
-				Returns the normal of the intersecting object shape face containing the collision point.
+				Returns the normal of the intersecting object's shape at the collision point.
```

Comment:
I'm new to Godot, so I may be wrong, but removing the word face may take away some clarity.  A normal is generally associated with one face. If so, perhaps:
> Returns the normal of the face of the intersecting object at the collision point.

Reply:
I'm open to it.  "shape face" just struck me as off. Too many uses of "of the", though.

> Returns the intersecting object's shape's normal at the collision point.

Reply:
I understand.  I had the same struggle as I was writing mine.  Yours feels less verbose, but perhaps not as specific.